### PR TITLE
Adds GitHub Actions

### PR DESCRIPTION
**Pull request by yuri-zubov**
Hi, @abrom !

I have just added CI. Merge it, please
image

### CODE REVIEW

**Comment by yuri-zubov** [comment]
@abrom what do think about this PR?

File: .github/workflows/ruby.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+on:
+  push:
+    branches: [ "master" ]
+  pull_request:
+    types: [ opened, synchronize, reopened, edited, ready_for_review ]
+
+jobs:
+  build:
+    name: Build + Publish
```
Comment (abrom) [inline .github/workflows/ruby.yml]: Should be just `Build`. 
```suggestion
    name: Build
```

File: .github/workflows/ruby.yml [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: write
```
Comment (abrom) [inline .github/workflows/ruby.yml]: This should be removed.

File: .github/workflows/ruby.yml [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+      matrix:
+        ruby-version: ['2.7', '3.0', '3.1']
+
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up Ruby
+        # To automatically get bug fixes and new Ruby versions for ruby/setup-ruby,
+        # change this to (see https://github.com/ruby/setup-ruby#versioning):
+        # uses: ruby/setup-ruby@v1
+        uses: ruby/setup-ruby@ee2113536afb7f793eed4ce60e8d3b26db912da4 # v1.127.0
```
Comment (abrom) [inline .github/workflows/ruby.yml]: Not sure why you're pegging this to v1.127.0? Latest using `ruby/setup-ruby@v1` would be v1.146.0
```suggestion
        uses: ruby/setup-ruby@v1
```

**Review by abrom — CHANGES_REQUESTED** [review]
A few things that will need to be tidied up, but otherwise looks good. Thanks @yuri-zubov

**Comment by yuri-zubov** [comment]
@abrom I fixed your comments! Recheck again, please

**Review by abrom — APPROVED** [review]
Looks good @yuri-zubov thanks!

**Pull request merged**